.storybook/_container.scss

```diff
@@ -1,4 +1,4 @@
 $css--font-face: true;
 $css--reset: true;
 
-@import '~carbon-components/scss/globals/scss/styles.scss';
+@import 'node_modules/carbon-components/src/globals/scss/styles';
```

Comment:
Is this something that we'll need to change after PR reviews or is this a permanent change?

Reply:
Oh, good catch, I'll swap it back before we merge.